File: DDS_Restaurant_Solution/DDS_Restaurant_VB/Models/Pedido.vb
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.ComponentModel.DataAnnotations
Public Class Pedido
    <Key>
    Public Property idPedido As Integer
    Public Property fecha As DateTime
    <MaxLength(10)>
    Public Property estadoPedido As String
    Public Property numIdentidad As String
    <ForeignKey("numIdentidad")>
    Public Property Cliente As Cliente
    Public Property idMesa As Integer
    <ForeignKey("idMesa")>
    Public Property Mesa As Mesa
End Class

File: DDS_Restaurant_Solution/DDS_Restaurant_Solution/Models/Pedido.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DDS_Restaurant_Solution.Models
{
    public class Pedido
    {
        [Key,DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int idPedido { get; set; }
        
        public DateTime fecha { get; set; }

        [MaxLength(10)]
        public string estadoPedido { get; set; }
        
        public string numIdentidad { get; set; }
        [ForeignKey("numIdentidad")]
        public Cliente Cliente { get; set; }

        public int idMesa { get; set; }
        [ForeignKey("idMesa")]
        public Mesa Mesa { get; set; }

        public int idUsuario { get; set; }
        [ForeignKey("idUsuario")]
        public Usuario Usuario { get; set; }
    }
}
